doc/ref/fpsemi.xml

```diff
@@ -34,12 +35,12 @@ Finitely presented monoids are obtained by factoring a free monoid by
 a set of relations, i.e. a set of pairs of words in the free monoid. 
 <P/>
 <Example><![CDATA[
-gap> f:=FreeMonoid("a","b");;
-gap> x:=GeneratorsOfMonoid(f);
+gap> f := FreeMonoid( "a", "b" );;
+gap> x := GeneratorsOfMonoid(f);
```

Comment:
I don't know if they are adhered to or not, but in this example whitespace is introduced inconsistently. This is probably not very important, but it'd be nice if it was done consistently.